crap, classloading issue :/

```
Caused by: java.util.concurrent.CompletionException: java.lang.IllegalAccessError: class 
  io.strimzi.test.container.StrimziKafkaResource tried to access method 'void 
  io.strimzi.test.container.StrimziKafkaContainer.<init>(java.lang.String)' 
  (io.strimzi.test.container.StrimziKafkaResource is in unnamed module of loader 
  io.quarkus.bootstrap.classloading.QuarkusClassLoader @403364e9; 
  io.strimzi.test.container.StrimziKafkaContainer is in unnamed module of loader 
  io.quarkus.bootstrap.classloading.QuarkusClassLoader @d400943)
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
```

--- a/301-quarkus-vertx-kafka/src/test/java/io/quarkus/qe/kafka/resources/StrimziKafkaResource.java
+++ b/301-quarkus-vertx-kafka/src/test/java/io/quarkus/qe/kafka/resources/StrimziKafkaResource.java
@@ -6,24 +6,25 @@ import java.util.Objects;
 import java.util.stream.Stream;
 
 import org.jboss.logging.Logger;
-import org.testcontainers.containers.Network;
 import org.testcontainers.lifecycle.Startables;
 
 import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
-import io.strimzi.test.container.StrimziKafkaContainer;
+import io.strimzi.test.container.StrimziKafkaCluster;
 
 public class StrimziKafkaResource implements QuarkusTestResourceLifecycleManager {
 
     private static final Logger LOG = Logger.getLogger(ConfluentKafkaResource.class);
 
-    private StrimziKafkaContainer kafkaContainer;
+    private StrimziKafkaCluster kafkaContainer;
     private SchemaRegistryContainer schemaRegistry;
 
     @Override
     public Map<String, String> start() {
-        Network network = Network.newNetwork();
 
-        kafkaContainer = new StrimziKafkaContainer("quay.io/strimzi/kafka:latest-kafka-4.1.0").withNetwork(network);
+        kafkaContainer = new StrimziKafkaCluster.StrimziKafkaClusterBuilder()
+                .withImage("quay.io/strimzi/kafka:latest-kafka-4.1.0")
+                .withSharedNetwork()
+                .build();
         schemaRegistry = new SchemaRegistryContainer("quay.io/apicurio/apicurio-registry-mem", "2.4.2.Final", 8080);
 
         Startables.deepStart(Stream.of(kafkaContainer, schemaRegistry)).join();

